src/main/java/buildcraftAdditions/armour/ItemKineticBackpack.java

```diff
@@ -19,10 +20,12 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class KineticBackpack extends ItemArmor implements IEnergyContainerItem {
+public class ItemKineticBackpack extends ItemArmor implements IEnergyContainerItem {
```

Comment:
Sorry, I always forget of the Jenkins. I think that everybody has pushed their changes...